test/e2e/beat/config.go

```diff
@@ -196,6 +196,7 @@ processors:
   hostPID: true  # Required by auditd module
   dnsPolicy: ClusterFirstWithHostNet
   hostNetwork: true
+  automountServiceAccountToken: true
```

Comment:
This is only necessary on some versions from your description, correct? Is it worth adding a comment here and updating the recipes/docs as well with this info? We call out other OpenShift specific configs

Reply:
@anyasabo, @sebgl: This is not OpenShift specific actually, only Beat version specific. I've added a comment, but I'd rather not add too much in the docs/samples for couple of reasons:
- if we do it for one setting, why we don't do it for all of them? it would a bit of work
- our docs start becoming Beats on k8s docs instead of just ECK docs
- we use those configs to test across all versions/environments, so we might end up with if statements in the comments :) we already turn off some things for old kind version, because it doesn't work there

In general I'd like to think about us testing Beats as a proof we can deploy them correctly and they work to some degree. I believe high fidelity documentation about what and why could be a rabbit hole for us and should live in Beat docs. WDYT?

Reply:
Ah gotcha my reading comprehension is poor. I think it's a blurry line for when it is useful to have beats documentation in our repo, but am good with your decision here 👍